hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1403,7 +1403,7 @@ protected void commitInternal(String instantTime, Map<String, HoodieData<HoodieR
       metadataMetaClient.reloadActiveTimeline();
     }
 
-    writeClient.startCommitWithTime(instantTime);
+    writeClient.startCommitForMetadataTable(instantTime, HoodieActiveTimeline.DELTA_COMMIT_ACTION);
```

Comment:
Let's create a separate JIRA to revisit the `metaClient` usage here.  Ideally `metadataMetaClient` should be reused, which has been refreshed before this statement. `startCommitWithTime` creates a new meta client inside; not sure if that's intentional.

Reply:
cc @nsivabalan

Reply:
This is an easy enough change to make in this PR since the variable is always available to us. Let me add it so we don't have to revisit this in a few days.

Reply:
Sg

Reply:
looks good to me.